### ISSUE TITLE
Support of Boxcryptor files: .bc

### ISSUE THREAD

**Issue by blandyuk** [hashcat/hashcat#3591]
The Boxcryptor files have a ".bc" file extention and start with the bytes "bc01".

The files do contain a JSON area which relates to the hash password using AES along with the keysize, blocksize, IV, etc. Presume this can be used to generate a hash to be used in hashcat.

Example file here:
https://anonfiles.com/jbG4UaT7y9/test_file.txt.bc_7z

I can provide account password used on request to try and verify if that's all that was used, but could be more complex depending on what was actually used. I find it odd they have this in each file, as what happens when a user changes their password? Might be using a separate key, but not sure.

Many thanks.

**Comment by magnumripper** [comment]
For what it's worth: Boxcryptor was acquired by Dropbox so (I assume) the file format is phased out as contracts expire.

https://www.boxcryptor.com/en/blog/post/new-chapter-with-dropbox/

**Comment by blandyuk** [comment]
Yeah, I have some .bc files, but looks like it's not going to be possible. No matter.

**Comment by TomStokes14** [comment]
I wouldn't mind taking a look! if you can send me with those details you mentioned I can try and see if there's something :)

**Comment by blandyuk** [comment]
Hi, example file here:
https://anonfiles.com/jbG4UaT7y9/test_file.txt.bc_7z

I can give account details, but would need your email if possible.

**Comment by ndv** [comment]
If there's a test bc file with a known password, please send it over to contact at dmitrynegoda.com along with the password. I see what I can do. No guarantee, since it is not open source.
